### ISSUE TITLE
Bug: Update dropped strains file to list accession instead of strain

### ISSUE THREAD

**Issue by j23414** [nextstrain/dengue#24]
### Current Behavior

Currently, strains listed in [phylogenetic/config/dropped_strains.txt](https://github.com/nextstrain/dengue/blob/5a61db5408de6fd698e16268126c548d731fc8ab/phylogenetic/config/dropped_strains.txt) are not being dropped since https://github.com/nextstrain/dengue/commit/8ab810f10cc64e45a9c2baf1bf6df9d6561578bc

### Expected behavior

Strains listed in `dropped_strains.txt` are not in the final phylogenetic tree.

### Possible solution

Perhaps cherry pick a commit like:

* https://github.com/nextstrain/dengue/commit/67016d1679a01069568d0fdae6b55cc43502d5b0

### Your environment: if browsing Nextstrain online
 - Operating system:
 - Browser:

### Your environment: if running Nextstrain locally
 - Operating system:
 - Browser:
 - Version (e.g. `auspice 2.7.0`):

**Comment by victorlin** [comment]
Good catch! If I understand correctly, `phylogenetic/config/dropped_strains.txt` is used for `augur filter --exclude` so it should be updated alongside 8ab810f10cc64e45a9c2baf1bf6df9d6561578bc. Looking more carefully at #12, the `--sequences` input should also be updated to match the new ID values but I don't see that it was changed. Does it still work?

**Comment by j23414** [comment]
Correct, and yes the `--sequences` input still works :D 

```bash
git clone https://github.com/nextstrain/dengue.git
cd dengue/phylogenetic
nextstrain build . data/sequences_all.fasta
grep  ">" data/sequences_all.fasta | head -n5
```

Which shows the sequences are ID'd by accession, not strain name:

```
>NC_075403
>NC_075435
>OQ919688
>ON123563
>ON123564
```

**Comment by victorlin** [comment]
Good to know! But how did it work before #12 if the sequences were ID'd by `accession` but `augur filter` was using `strain` as the ID column?

**Comment by j23414** [comment]
> augur filter was using strain as the ID column?

This worked when we were still pulling strains from fauna (which also took advantage of the deduplication of strain names of fauna). 

However, we shifted to using the `ingest` folder and pulling data using `ncbi datasets`. After ingest was merged, the [data.nextstrain.org/files](https://github.com/nextstrain/dengue/blob/5a61db5408de6fd698e16268126c548d731fc8ab/phylogenetic/rules/prepare_sequences.smk#L24-L25) were updated so we could have a smooth transition.
